BookReader/plugins/plugin.vendor-fullscreen.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Toggles browser's native fullscreen mode if available
+ */
+
+
+(function(BR) {
+  var event_namespace = ".bookreader_vendor-fullscreen";
+
+  jQuery.extend(BR.defaultOptions, {
+    enableVendorFullscreenPlugin: true
+  });
+
+  BR.prototype.setup = (function(super_) {
+    return function(options) {
+      super_.call(this, options);
+
+      this.isVendorFullscreenActive = false;
+    };
+  })(BR.prototype.setup);
+
+  BR.prototype.getInitialMode = (function(super_) {
+    return function(params) {
+      var nextMode = super_.call(this, params);
+      if (this.isVendorFullscreenActive) {
+        nextMode = this.constMode1up;
+      }
+      return nextMode;
+    };
+  })(BR.prototype.getInitialMode);
+
+  BR.prototype.init = (function(super_) {
+    return function() {
+      super_.call(this);
+
+      if (BR.util.fullscreenAllowed()) {
+        // In fullscreen mode, the colorbox and overlay need to be inside the
+        // fullscreen element to display properly.
+        BR.util.bindFullscreenChangeListener(this, function(e) {
+          e.data.resize();
+          e.data.updateBrClasses();
+          var cboxOverlay = $("#cboxOverlay");
+          var cbox = $("#colorbox");
+          if (BR.util.isFullscreenActive()) {
+            // In full screen mode, the colorbox and overlay need
+            // to be children of the fullscreen element to display properly.
+            var $fullscreen = $(BR.util.getFullscreenElement());
+            $fullscreen.append(cboxOverlay).append(cbox);
+          } else {
+            // In non-fullscreen mode, the colorbox and overlay need
+            // to be children of the main document body.
+            $(document.body).append(cboxOverlay).append(cbox);
+          }
+        });
+      }
+    }
+  })(BR.prototype.init);
+
+  BR.prototype.enterFullWindow = function() {
+    this.refs.$brContainer.css('opacity', 0);
+
+    var windowWidth = $(window).width();
+    if (windowWidth <= this.onePageMinBreakpoint) {
+      this.switchMode(this.constMode1up);
+    }
```

Comment:
Is this the same "bug" from #134 ?

Reply:
I think this was the original author's way of changing the behavior in the same way that my fix did, where I just removed the check entirely.